src/makeRobot.js

```diff
@@ -39,6 +39,70 @@
  */
 function makeRobot(name, wheels, version) {
   // write code here
+  return {
+    name,
+    wheels,
+    version,
+
+    coords: {
+      x: 0,
+      y: 0,
+    },
+
+    get info() {
+      return `name: ${this.name}, chip version: ${this.version},`
+      + ` wheels: ${this.wheels}`;
+    },
+
+    get location() {
+      return `${this.name}: x=${this.coords.x}, y=${this.coords.y}`;
+    },
+
+    goForward(steps = 1) {
+      if (steps < 0) {
+        return this;
+      }
+
+      this.coords.y += steps;
+
+      return this;
```

Comment:
i think beter to check that steps > 1,because you can remove one return from this function example:
```suggestion
     if (steps > 0) {
        this.coords.x -= steps;
      }

      return this;
```